File: backend/app/Models/FirstAid.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Model;
use Illuminate\Database\Eloquent\Factories\HasFactory;

class FirstAid extends Model
{
    use HasFactory;

    protected $fillable = [
        'title',
        'content',
        'icon',
        'priority_order',
        'is_active'
    ];

    protected $casts = [
        'is_active' => 'boolean',
        'priority_order' => 'integer'
    ];
} 

File: app/lib/src/models/first_aid.dart
class FirstAid {
  final int id;
  final String title;
  final String content;
  final String icon;
  final int priorityOrder;
  final bool isActive;

  FirstAid({
    required this.id,
    required this.title,
    required this.content,
    required this.icon,
    required this.priorityOrder,
    required this.isActive,
  });

  factory FirstAid.fromJson(Map<String, dynamic> json) {
    return FirstAid(
      id: json['id'],
      title: json['title'],
      content: json['content'],
      icon: json['icon'] ?? '',
      priorityOrder: json['priority_order'] ?? 0,
      isActive: json['is_active'] ?? true,
    );
  }
} 